plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -427,6 +427,19 @@ public void testClusteringKeyOnlyPushdown()
         assertEquals(execute(sql).getRowCount(), 1);
     }
 
+    @Test
+    public void testNotEqualPredicateOnClusteringColumn()
```

Comment:
Is the original bug expose-able with `io.trino.testing.AbstractTestDistributedQueries#testDataMappingSmokeTest`?

(perhaps no, because we don't create clustering columns there, do we?)

Reply:
No because WHERE statement in Cassandra needs to be composed of `of relations on the columns that are part of the PRIMARY KEY and/or have a secondary index defined on them.`